test/unit/pipeline/number.ts

```diff
@@ -0,0 +1,26 @@
+/*
+	File: test/unit/pipeline/number.ts
+	cpuabuse.com
+*/
+
+/**
+ * Tests `number`.
+ */
+
+import { strictEqual } from "assert";
+import { processNumber } from "../../../src/pipeline/number";
+//@ts-ignore
+import { food as input } from "../../input/pipeline";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Imports should be sorted alphabetically.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5058624)